Doc/library/stdtypes.rst

```diff
@@ -2636,6 +2636,15 @@ binary data. These restrictions are covered below.
    Using these ASCII based operations to manipulate binary data that is not
    stored in an ASCII based format may lead to data corruption.
 
+
+The following method is only for bytearray objects.
+
+.. method:: bytearray.copy()
+
+   Return a copy of the bytearray.  (Immutable bytes never need to be
+   copied.)
```

Comment:
Sorry if I'm being a pedant here. On first read I didn't immediately understand what the parenthesized sentence meant. After more thought, I'm guessing you were referring to the bytes object.

```suggestion
   Return a copy of the bytearray.  (Byte objects are immutable and never need
   to be copied.)
```

Reply:
Thank you.  The doc should be clear to people who are not 20+ year veterans of Python ;-).  I left this open after CI passed to allow for random review.  I wish more newer contributors should do them.